code/modules/admin/verbs/playsound.dm

```diff
@@ -52,6 +56,56 @@
 	playsound(get_turf(src.mob), S, 50, 0, 0)
 	SSblackbox.add_details("admin_verb","Play Local Sound") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/play_web_sound()
+	set category = "Fun"
+	set name = "Play Internet Sound"
+	if(!check_rights(R_SOUNDS))
+		return
+
+	if(!config.invoke_youtubedl)
+		to_chat(src, "<span class='boldwarning'>Youtube-dl was not configured, action unavailable</span>") //Check config.txt for the INVOKE_YOUTUBEDL value
+		return
+
+	var/web_sound_input = input("Enter content URL (supported sites only, leave blank to stop playing)", "Play Internet Sound via youtube-dl") as text|null
+	if(istext(web_sound_input))
+		var/web_sound_url = ""
+		var/pitch
+		if(length(web_sound_input))
+
+			web_sound_input = trim(web_sound_input)

```

Comment:
trimming should be done before checking length

Reply:
I did this on purpose so that ONLY if the input is completely blank will it stop audio